docs/iris/src/userguide/saving_iris_cubes.rst

```diff
@@ -106,3 +107,32 @@ Such a custom saver will need be written to meet the needs of the file format an
 
 Implementing a bespoke saver is out of scope for the user guide.
 
+Caching cubes and cube lists to pickle files
+--------------------------------------------
+
+It should always be possible to create a temporary cache file containing a cube or cube list using the Python `Pickle <https://docs.python.org/2/library/pickle.html>`_ functionality. This can be useful when the cube or cube list has been lazily loaded so the pickle file itself will contain only a reference to the data in the original files. In this state writing, and subsequently reading, a pickle file is very fast.
```

Comment:
`cube` and `cube list` are both class objects that sphinx can get to the documentation for. They also have standardised names within the Iris documentation; being `Cube` and `CubeList` respectively.

As such, you should in all cases refer to them as follows:

* ```:class:~`iris.cube.Cube` ```
* ```:class:~`iris.cube.CubeList` ```